odxtools/basicstructure.py

```diff
@@ -224,7 +225,7 @@ def convert_bytes_to_physical(self,
 
         return inner_decode_state.parameter_values, decode_state.cursor_position + inner_decode_state.cursor_position
 
-    def encode(self, coded_request: Optional[bytes] = None, **params) -> bytes:
+    def encode(self, coded_request: Optional[bytes] = None, **params: Any) -> bytes:
```

Comment:
Sure about Any?

Reply:
That was a quick and dirty way to get around mypy errors: for the `kwargs` dict, mypy seems to assume that the type of the annotation is used for *all* keyword arguments, so just setting this to `Any` always works.

In this place, using `ParameterValue` would be better, though (it is almost the same as `Any` by now, though). fixed, thanks: 9c7ecc0818b4cb4b9eedde9b13f3eb8599f78eae